Documentation/capabilities-guide.md

```diff
@@ -222,6 +223,101 @@ root
 nc: bind: Permission denied
 ```
 
+## Overriding capabilities
+
+Capabilities sets are typically defined when creating images, as they are tightly
+linked to specific app requirements. However, image consumers may need to further
+tweak/restrict the set of available capabilities in specific local scenarios.
+This can be done either by permanently patching the manifest of specific images, or
+by overriding capability isolators via command line arguments.
+
+### Patching images
+
+Image manifests can be manipulated manually, by unpacking the image and editing
+the manifest file, or via helper tools like `actool`.
+In order to override pre-defined capabilities set, it is enough to replace any
+existing capability isolators with a new custom one.
+
+`actool patch-manifest` provides two options to manipulate the retain set
+(`-capability`) and the remove set (`-revoke-capability`).
+Starting from the `caps-retain-set-example.aci` image above, it is possible to
+transform its capabilities set as follows:
+
+```
+$ actool cat-manifest caps-retain-set-example.aci
+...
+    "isolators": [
+      {
+        "name": "os/linux/capabilities-retain-set",
+        "value": {
+          "set": [
+            "CAP_NET_RAW"
+          ]
+        }
+      }
+    ]
+...
+
+$ actool patch-manifest -capability CAP_NET_RAW,CAP_NET_BIND_SERVICE caps-retain-set-example.aci caps-retain-set-patched.aci
+
+$ actool cat-manifest caps-retain-set-patched.aci
+...
+    "isolators": [
+      {
+        "name": "os/linux/capabilities-retain-set",
+        "value": {
+          "set": [
+            "CAP_NET_RAW",
+            "CAP_NET_BIND_SERVICE"
+          ]
+        }
+      }
+    ]
+...
+
+```
+
+And then verify that the additional `CAP_NET_BIND_SERVICE` capability is now
+retained:
+
+```
+$ sudo rkt run --interactive --insecure-options=image caps-retain-set-patched.aci
+image: using image from file stage1-coreos.aci
+image: using image from file caps-retain-set-patched.aci
+image: using image from local store for image name quay.io/coreos/alpine-sh
+
+/ # nc -v -l -p 80
+listening on [::]:80 ...
+```
+
+### Overriding capabilities at run-time
+
+Capabilities can be directly overridden at run-time from the command-line,
+without the need of manipulating images.
+`rkt run` provides two options to manipulate the retain set
+(`--cap-retain`) and the remove set (`--cap-remove`).
+
+For example, capability isolators can be added via command-line at execution
```

Comment:
Capabilities isolators can be added on the command line at run time by specifying the desired overriding set, as shown in this example: